.gitignore

```diff
@@ -5,3 +5,6 @@ node_modules/
 debug
 .vagrant
 /static/
+loaddumps
+sqldumps.py
+.hurumapKEenv
```

Comment:
@tinamorale What's the `.hurumapKEenv` file?

Also, should we ignore the `.vscode*` files + folders?